book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    rental = Rental.new(date, self, person)
+    @rentals << rental
+  end
```

Comment:
- [ ] Creating the rental on line 11 adds the rental to the array so we don't need line 12. It will create duplicate

  ```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
  ```